plugins/modules/zammad_ticket.py

```diff
@@ -0,0 +1,318 @@
+#!/usr/bin/python
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+author:
+- Scaleup Technologies
```

Comment:
Please don't list this separately, rather
- Melvin Ziemann, ScaleUp Technologies GmbH & Co. KG (@cloucs)